### PR TITLE
move docs out of top-level

### PR DESCRIPTION
**Pull request by samiebee43**
There should not be more than one README file. The README can link to other documents.

### CODE REVIEW

**Review by mzieg — CHANGES_REQUESTED** [review]
Should there be updates in README.md?

https://github.com/WasatchPhotonics/ENLIGHTEN/tree/sb-doc#table-of-contents

**Comment by samiebee43** [comment]
Yes and to other files as well. Added.

File: docs/CONTENTS.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Major directories and files in the source distribution:
         - environments\
             - Anaconda environment files for supported developer platforms
     - Windows-only
-        - bootstrap.bat (see [Windows](README_WINDOWS.md))
```
Comment (mzieg) [inline docs/CONTENTS.md:50]: Should we merge BUILD_WIN10 and _WIN11 into a file by this name?

File: docs/MAINTENANCE.md [hunk 2]
```diff
@@ -22,16 +22,16 @@ name the file after the class for consistency.
 
 - update version number in enlighten/common.py
     - re-run Enlighten.py (forces common.pyc recompile)
-- update README\_CHANGELOG.md 
-- build a Windows installer (see [Windows](README_WINDOWS.md))
+- update CHANGELOG.md 
+- build a Windows installer (see [Windows](BUILD_WINDOWS.md))
```
Comment (mzieg) [inline docs/MAINTENANCE.md:26]: see above (no BUILD_WINDOWS or WINDOWS)

**Review by mzieg — APPROVED** [review]

File: docs/CONTENTS.md (hunk 1 above)
Comment (samiebee43) [reply docs/CONTENTS.md:50]: We should just link to one of them. Originally Win10 was the main build instructions and Win11 [just redirected to Win10](https://github.com/WasatchPhotonics/ENLIGHTEN/wiki/Build-Win11).

But recently I was able to make a build from Windows 11 using almost the exact same steps. (Slight tweak to Bootstrap.bat #344)

**Pull request merged**